src/ftof.cpp

```diff
@@ -0,0 +1,59 @@
+/*! \file ftof.cpp
+ *
+ * \brief Contains functions to transfer EigenMatrix and EigenVector to FASP
+ *  or to tranfer FASP to Fenics
+ */
+
+#include "ftof.h"
+
+using namespace dolfin;
+
+// Link the dolfin::EigenMatrix Mat_A to dCSRmat format from FASP
+void EigenMatrixTOdCSRmat( dCSRmat* dCSR_A, const EigenMatrix* mat_A)
+{
+  int i,j,k=0;
+  int nrows = mat_A->size(0);
+  int ncols = mat_A->size(1);
+  int nnz = mat_A->nnz();
+  double* vals;
+  int *JA;
+  int *el_number; // el_number refer the index in val of the first element in each row
+  int *IA;
+
+  el_number= (int *)get<0>(mat_A->data());
+  JA = (int *)get<1>(mat_A->data());
+  vals = (double *)get<2>(mat_A->data());
```

Comment:
when we access methods from libraries, we should prefix them to avoid using the wrong function from a different library.
e.g. 
``el_number= (int *)boost::tuples::get<0>(mat_A->data());``